oe_theme.libraries.yml

```diff
@@ -36,18 +25,6 @@ context_nav:
   dependencies:
     - core/drupal
     - oe_theme/base
-file_translations:
-  js:
-    js/file_translations.js: {}
-  dependencies:
-    - core/drupal
-    - oe_theme/base
-inpage_navigation:
-  js:
-    js/inpage_navigation.js: {}
-  dependencies:
-    - core/drupal
-    - oe_theme/base
 lang_select_pages:
```

Comment:
The js was removed but the library is still here.

Reply:
Also its still used on links--oe-multilingual-content-language-block.html.twig

Reply:
Very good catch.